tox.ini

```diff
@@ -7,14 +7,30 @@ skip_install = true
 setenv =
     VIRTUALENV_NO_DOWNLOAD=1
 
+[testenv:lint-shell]
+whitelist_externals = bash
+basepython = python3.6
+skip_install = true
+deps =
+    -r{toxinidir}/lint-requirements.txt
```

Comment:
Now you're installing all lint deps in 3 different venvs. Either

- List linter-specific deps here (I think that'd be my preference)
- Set up Tox to re-use the same env between the 3 jobs

Reply:
lint deps are minimal, so linter-specific it is.